terraform/eks-node-groups/single-az-node-groups/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "node_groups" {
+  description = "Outputs from EKS node groups. Map of maps, keyed by `var.node_groups` keys. See `aws_eks_node_group` Terraform documentation for values"
+  value       = aws_eks_node_group.workers
+}
+
+output "aws_auth_roles" {
```

Comment:
Is this actually needed or just a copypasta?  I'm wondering how complex the usage is if you constantly need to modify aws auth

Reply:
We don't really need this output I believe.

Reply:
ok cool, it's fine to keep I just wanted to get a feel for its usability